### ISSUE TITLE
Kriby 2.2 update breaks accordions

### ISSUE THREAD

**Issue by mearno** [ljacobs-sml/kirby-accordionHeadlineField#6]
With the new panel structure in Kirby 2.2. the accordions won't work anymore. See screenshot.

![schermafbeelding 2015-11-19 om 09 08 17](https://cloud.githubusercontent.com/assets/2323826/11265548/4bfaab8c-8e9d-11e5-803c-ae6bf6f04be2.png)

**Comment by crispconcepts** [comment]
This should be working now. Please let me know if you're still seeing any issue.

**Comment by mearno** [comment]
Thanks.

The first panel is opened by default so I did:

}); //.first().addClass('active').parent().next().addClass('active').slideDown(0); 

Not sure if that has further consequences though but it seems to work :)

**Comment by mearno** [comment]
I also run into the following:

Inside a panel I use a structured field.
When I add a new field and cancel, the headers stop working(won't open). (probably a js thing)
When I close the page and open it again the headers work fine.

**Comment by crispconcepts** [comment]
Hmm, underneath the body element, there's a div class="main" that on ajax calls seems to get id="null" appended to it. On save, the id="null" goes away on it's own. On cancelling the structure id="null" remains.

To fix the on save issue, I wrapped the code in a function and triggered it again on ajaxSuccess. I tried doing the same on ajaxError and didn't work. I wonder i there's a deeper issue. I don't think I'll get to this for a little while as I'm very busy at work and with the holidays, but I will look into debugging it further (unless someone else wants to) and bringing any issues I find to bastian's attention.
